astroquery/ipac/irsa/most/core.py

```diff
@@ -0,0 +1,1017 @@
+            containing the links to image and region files and minimum object
+            metadata, while `metadata` contains the image metadata and object
+            positions. The `region` key contains a link to the DS9 region file
+            representing the matched object trajectory and search boxes. When
+            existing, `fits_tarball` and `region_tarball` are links to the
+            tarball archives of the fits and region images.
+        """
+        retdict = {}
+        html = BeautifulSoup(response.content, "html5lib")
+        download_tags = html.find_all("a", string=re.compile(".*Download.*"))
+
+        # this is "Download Results Table (above)"
+        results_response = self._request("GET", download_tags[0]["href"])
+        retdict["results"] = Table.read(results_response.text, format="ipac")
+
+        # this is "Download Image Metadata with Matched Object position Table"
+        imgmet_response = self._request("GET", download_tags[1]["href"])
+        retdict["metadata"] = Table.read(imgmet_response.text, format="ipac")
+
+        # this is "Download DS9 Region File with the Orbital Path", it's a link
+        # to a DS9 region file
+        # regions_response = self._request("GET", download_tags[2]["href"])
+        retdict["region"] = download_tags[2]["href"]
+
+        if withTarballs:
+            retdict["fits_tarball"] = download_tags[-1]["href"]
+            retdict["region_tarball"] = download_tags[-2]["href"]
+
+        return retdict
+
+    @class_or_instance
+    def list_catalogs(self):
+        """Returns a list of queriable catalogs."""
+        response = self._request("GET", conf.interface_url, timeout=self.TIMEOUT)
+
+        html = BeautifulSoup(response.content, "html5lib")
+        catalog_dropdown_options = html.find("select").find_all("option")
+
+        catalogs = [tag.string for tag in catalog_dropdown_options]
+        # I think I saw somewhere that some password prompt will pop up for
+        # catalogs listed as '--- Internal use only:' but there are seemingly
+        # no limits to queries there.
+        if "--- Internal use only:" in catalogs:
+            catalogs.remove("--- Internal use only:")
+        return catalogs
+
+    def get_images(
+            self,
+            catalog="wise_merge",
+            input_mode="name_input",
+            ephem_step=0.25,
+            obs_begin=None,
+            obs_end=None,
+            obj_name=None,
+            obj_nafid=None,
+            obj_type=None,
+            mpc_data=None,
+            body_designation=None,
+            epoch=None,
+            eccentricity=None,
+            inclination=None,
+            arg_perihelion=None,
+            ascend_node=None,
+            semimajor_axis=None,
+            mean_anomaly=None,
+            perih_dist=None,
+            perih_time=None,
+            get_query_payload=False,
+            save=False,
+            savedir='',
+    ):
```

Comment:
I really don't like black's love of whitespace and this way of 20+ lines instead of ~4 for the signature. Can we put some sense back in it, into ~100 char long lines instead?